.github/workflows/wheels.yml

```diff
@@ -0,0 +1,76 @@
+# Workflow to build and test wheels
+name: Wheel builder
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    tags:
+      # Stable release build
+      - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
also include `post` and `rc` builds? (I'm not entirely certain how this is supposed to work)

Reply:
I had not thought about it, but it is quite interesting. I will add the patterns to build `post` and `rc` builds.

Thanks @adrinjalali!